frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -208,6 +211,10 @@ const HostSummary = ({
         : REFETCH_TOOLTIP_MESSAGES[hostMdmDeviceStatus];
     }
 
+    if (isIosOrIpadosHost) {
```

Comment:
Nice to have, not blocking – might as well save a few lines of evaluation and put the `null` case right at the top of the function